internal/tiltfile/local_resource.go

```diff
@@ -63,12 +64,14 @@ func (s *tiltfileState) localResource(thread *starlark.Thread, fn *starlark.Buil
 	if err := s.unpackArgs(fn.Name(), args, kwargs,
 		"name", &name,
 		"cmd?", &updateCmdVal,
+		"env?", &updateEnv,
```

Comment:
i think these need to get added to the end of the arg list, so that these don't break people using positional args)

the starlark folks have been receptive to having functions that only accept named args (not positional args), but that's for a future compatibility-breaking release! https://github.com/google/starlark-go/issues/249

Reply:
D'oh - you're absolutely right; moved them to where they should be in f20723045a7eeb7e590529ce2ae887882946fbb5